tools/rvn/rvn.csproj

```diff
@@ -119,9 +115,11 @@
     <PackageReference Include="DasMulli.Win32.ServiceUtils.Signed" Version="1.1.0" />
     <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.1.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Portable.BouncyCastle" Version="1.9.0" />
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="6.0.0" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="6.0.0" />
+    <ProjectReference Include="..\..\src\Raven.Server\Raven.Server.csproj" />
```

Comment:
Referencing entire Raven.Server project isn't the way to go. See that we were _linking_ just the files that we need e.g.:

```
<Compile Include="..\..\src\Raven.Server\Commercial\SetupMode.cs" Link="Server\Commercial\SetupMode.cs" />
```

Now you dropped all of those and referenced _everything_.  Linking just the necessary files was the right thing to do here.